types.js

```diff
@@ -0,0 +1,61 @@
+// @flow
```

Comment:
I copied these from https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/eslint with some modifications.  I'll add a comment given credit.  It would be nice if we could use TypeScript for these actions, then we'd be able to get types for `eslint` and `@actions/core` and other packages without a lot of work.